### ISSUE TITLE
FEAT: Press "Enter" to save note rather than "Command+Enter"

### ISSUE THREAD

**Issue by VarfajMarkus** [iAmKevinCruz/raycast-timecamp-extension#14]
Simplify UX, fewer keyboard shortcuts for similar interactions. "Enter" to enter note, "Enter" to save note and close window.

**Comment by iAmKevinCruz** [comment]
I don't think that is possible accosding to the docs below. I may be able to add a listener inside the form field for `ENTER` and then manually submit the form. I'll try that and let you know how that goes.

https://developers.raycast.com/api-reference/keyboard
